README.md

```diff
@@ -106,6 +106,30 @@ Currently, we support the following types of assets:
 - [ERC-721](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/) (Non-fungible tokens)
 - [ERC-1155](https://ethereum.org/en/developers/docs/standards/tokens/erc-1155/) (Multi-tokens)
 
+### Input encodings for deposits
+
+As explained above, when a portal teleports assets to the execution layer, it adds an input to the DApp's input box, which will then need to be interpreted and validated by the off-chain machine. In order to do that, the machine will need to understand and decode the input's payload.
+
+The input payloads for deposits are always specified as packed ABI-encoded parameters, as detailed below:
+
+| Asset | Payload |
+| :- | :- |
+| Ether | `address sender, uint256 value, bytes execLayerData` |
+| ERC-20 | `bool success, address token, address sender, uint256 amount, bytes execLayerData` |
+| ERC-721 | `address token, address sender, uint256 tokenId, bytes data` `*` |
+| ERC-1155 (single) | `address token, address sender, uint256 tokenId, uint256 value, bytes data` `*`|
+| ERC-1155 (batch) | `address token, address sender, bytes data` `**` |
+
+**`*`** The ERC-721 and ERC-1155 (single) `data` fields correspond to the ABI-encoding of: `bytes baseLayerData, bytes execLayerData`
+
+**`**`** The ERC-1155 (batch) `data` field corresponds to the ABI-encoding of: `uint256[] tokenIds, uint256[] values, bytes baseLayerData, bytes execLayerData`
+
+As an example, the deposit of 100 wei (of Ether) sent by address `0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266` with data `0xabcd` would result in the following input payload:
```

Comment:
Wei is generally written capitalized.

Reply:
Done in https://github.com/cartesi/rollups/pull/42/commits/e4ef2ae0655fe2b21af3afedda7828fbdb4aab80